pegasus/emails/dashboard.html

```diff
@@ -0,0 +1,7 @@
+---
+from: '"Code.org" <noreply@code.org>'
+subject: "<%= subject %>"
+---
+<%= body %>
+
+<img src="<%= tracking_pixel %>"/>
```

Comment:
if unsubscribe affects dashboard emails we should probably expose that in the account settings page